src/littlefs.c

```diff
@@ -0,0 +1,110 @@
+#include "littlefs.h"
+#include "w25x.h"
+
+#define BLOCK_SIZE 4096
+#define BLOCK_COUNT 2048
+#define FLASH_SIZE (BLOCK_SIZE * BLOCK_COUNT)
+
+// 512 kB for dump (128 x 4kB)
+#define DUMP_BLOCK_COUNT 128
+
+// Address offset to skip area reserved for the dump
+#define ADDR_OFFSET (BLOCK_SIZE * DUMP_BLOCK_COUNT)
+
+lfs_t lfs;
```

Comment:
/suggestion/ It seems to me that this and all other definitions in this file are private to this file (except `littlefs_init`). Make them `static` in the case that is true

Reply:
Makes sense, I'm going to update.